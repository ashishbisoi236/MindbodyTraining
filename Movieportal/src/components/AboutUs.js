import React, { Component } from "react";

const styleObj = {
    color: "#4a54f1",
    textAlign: "center",
    paddingTop: "100px",
}

class AboutUs extends Component {
    
  render() {
    return (
      <>
        <h3 style={styleObj}>About me/ Us</h3>
      </>
    );
  }
}

export default AboutUs;
