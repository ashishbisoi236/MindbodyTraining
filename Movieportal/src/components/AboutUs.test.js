import React from 'react';
import renderer from "react-test-renderer";
import AboutUs from "./AboutUs";

test("Creating a snapshot of AboutUs component", () => {
    const component = renderer.create(<AboutUs />);
    const treesnapshot = component.toJSON();
    expect(treesnapshot).toMatchSnapshot();
});