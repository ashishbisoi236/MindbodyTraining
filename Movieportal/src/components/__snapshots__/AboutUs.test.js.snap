// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creating a snapshot of AboutUs component 1`] = `
<h3
  style={
    Object {
      "color": "#4a54f1",
      "paddingTop": "100px",
      "textAlign": "center",
    }
  }
>
  About me/ Us
</h3>
`;
